infra/README.md

```diff
@@ -0,0 +1,75 @@
+- It is based on the [Nava platform infrastructure template](https://github.com/navapbc/template-infra).
+- As currently deployed, this project has the following AWS environments:
+    - `dev`
+    - `staging`
+    - `prod`
+
+### 📂 Directory structure
+
+The infrastructure for this project has extended the Nava platform infrastructure template's [module architecture](../docs/infra/module-architecture.md). It currently looks like this:
+
+```text
+infra/                  Infrastructure code
+  accounts/             Root module for IaC and IAM resources
+  [app_name]/           Application directory: infrastructure for the main application
+  modules/              Reusable child modules
+  networks/             Account level network config (shared across all apps, environments, and terraform workspaces)
+```
+
+Each application directory contains the following:
+
+```text
+  app-config/         Application-level configuration for the application resources (different config for different environments)
+  build-repository/   Docker image repository for the application (shared across environments and terraform workspaces)
+  database/           Configuration for database (different config for different environments)
+  service/            Configuration for containers (load balancer, application service) (different config for different environments)
+```
+
+### 🏗️ Project architecture
+
+#### 🥞 Infrastructure layers
+
+The infrastructure template is designed to operate on different layers.
+
+- Account layer
+- Network layer
+- Build repository layer (per application)
+- Database layer (per application)
+- Service layer (per application)
+
+### 🔀 Project workflow
+
+This project relies on Make targets in the [root Makefile](../Makefile), which in turn call shell scripts in [./bin](../bin). The shell scripts call terraform commands. Many of the shell scripts are also called by the [Github Actions CI/CD](../.github/workflows).
+
+Backend configuration is saved as [`.tfbackend`](https://developer.hashicorp.com/terraform/language/settings/backends/configuration#file) files. Most `.tfbackend` files are named after the environment. For example, the `[app_name]/service` infrastructure resources for the `dev` environment are configured via `dev.s3.tfbackend`. Resources for a module that are shared across environments, such as the build-repository, use `shared.s3.tfbackend`. Resources that are shared across the entire account (e.g. /infra/accounts) use `<account name>.<account id>.s3.tfbackend`.
+
+Generally you should use the Make targets or the underlying bin scripts, but if you want to directly use the `terraform` cli, you need to pass in the `.tfbackend` file to init commands and the `.tfvars` file to other commands:
+
+```sh
+infra/portal/service$ terraform init -backend-config=dev.s3.tfbackend
+infra/portal/service$ terraform apply -var="environment_name=dev"
+```
```

Comment:
observation: these instructions duplicate https://github.com/navapbc/template-infra/blob/main/docs/infra/making-infra-changes.md

since using the terraform cli directly isn't the "recommended" option, perhaps we could hid the actual shell code snippet and just keep the summary about using the make targets. and then we could link to the doc for the in depths stuff. for example something like:

```suggestion
Generally you should use the Make targets or the underlying bin scripts. For other ways of making changes, including directly using the `terraform` cli, see [Making infra changes](../docs/infra/making-infra-changes.md)
```